src/Commands/LowGoalShoot.cpp

```diff
@@ -0,0 +1,11 @@
+#include "LowGoalShoot.h"
+#include "SetArms.h"
+#include "SetLifter.h"
+#include "ShooterControl.h"
+
+LowGoalShoot::LowGoalShoot()
+{
+  AddSequential(new SetArms(Intake::kUp));
+  AddParallel(new SetLifter(Shooter::kDown));
```

Comment:
The subsystem is included because we used a command that also uses the subsystem. The command includes the subsystem because that is how the interface is defined. 

Including the class that is needed is correct.